modules/ROOT/pages/sap/sap-connector-config-topics.adoc

```diff
@@ -0,0 +1,171 @@
+
+For this to work you must set the property name, as defined by SAP, in your configuration.
+
+=== SAP Inbound
+
+There are no special Advanced properties for this configuration.
+
+=== SAP Outbound
+
+[%header%autowidth.spread]
+|===
+|Field |XML Attribute |Description |Default Value
+|Display Name |name |The reference name of the endpoint used internally by Mule configuration. |
+|Encoding |encoding |Canonical name of an encoding. See https://docs.oracle.com/javase/7/docs/technotes/guides/intl/encoding.doc.html[Oracle's Supported Encodings]. |
+|Default idocument Version |defaultIdocumentVersion |This version is used when sending the IDoc. Values for the IDoc version correspond to IDOC_VERSION_xxxx constants in com.sap.conn.idoc.IDocFactory. |
+|Disable Function template cache flag |disableFunctionTemplateCacheFlag |Indicates if the function template cache should be disabled. |false
+|Evaluate response flag |evaluateResponseFlag |Indicates if an error response should throw an exception or let the user handle it as another result. |false
+|Log trace flag |logTraceFlag |Indicates if the trace should be logged on the Mule server. |false
+|Use SAP default format |useSAPFormat |Indicates to use SAP default format when working with BAPI functions over sRFC. |false
+|===
+
+== SAP JCo Notes
+
+SAP JCo facilitates communication between an SAP backend system and a Java application. SAP JCo allows Java programs to connect to SAP systems and invoke Remote Function Modules. SAP JCo also allows parsing of IDocs (SAP Intermediate Documents), among other object types. Both inbound and outbound communications are supported.
+
+* Java API: Handles dynamic metadata lookup and caching. It implements JCO.Function, which is the container for parameters and/or tables for the SAP Function Module (BAPI) in Java. Java applications are built on top of the Java API.
+
+* JNI (Java Native Interface): Originally, SAP created libraries in C language to allow direct RFC calls to SAP, to manipulate with data. JCo wraps C libraries in Java to provide platform-native access into the SAP system. RFC Middleware uses RFC Library through (JNI) Layer.
+
+* RFC (Remote Function Call): Communication with the SAP system is performed over the RFC protocol. RFC means calling BAPI or triggering IDoc processing that runs in another system as calling program. The RFC interface enables function calls between two SAP systems or between the SAP and external system.
+
+* RFC Library: Libraries of C language-based functions to access the SAP system. RFC library is addressed by JNI.
+
+* RFC Layer: SAP component that processes RFC calls.
+
+* SAP Java IDoc Class Library: Provides structured, high-level interpretation and navigation of SAP IDocs in Java. It consists of the following add-on packages:
+** SAP Java Base IDoc Class Library: a middleware-independent library that provides a set of general base classes and interfaces for middleware dependent Java IDoc Class Library implementations.
```

Comment:
Suggest formatting the sublist items in the same definition list style.

Reply:
Class Library: a middleware-independent
  >  Class Library: A middleware-independent

Reply:
middleware dependent Java IDoc
  >   middleware-dependent Java IDoc